editor/store/reducer.js

```diff
@@ -436,6 +438,7 @@ export const editor = flow( [
  */
 export function currentPost( state = {}, action ) {
 	switch ( action.type ) {
+		case 'SETUP_EDITOR_STATE':
```

Comment:
This is slightly concerning only because the logic handles both `action.post` and `action.edits`, where we very much rely on `action.post` to take effect in `SETUP_EDITOR_STATE` since `isEditedPostNew` selector uses the auto-draft status from there and not the initial overriding edits.

Reply:
But this is exactly how it worked before, the currentPost is till containing `auto-draft` while the edits have the `draft` change. Can't see what's the issue?

Reply:
Not really an issue, but caught my attention. Might be something we want to document and/or test, but can be done separately.